preprod.healdata.org/portal/gitops.json

```diff
@@ -39,10 +39,10 @@
           "tooltip": "Gen3 workspaces are secure data analysis environments in the cloud that can access data from one or more data resources."
         },
         {
-          "name": "Notebook Browser",
+          "name": "Static Notebooks",
```

Comment:
I think we also want to update the `resourceBrowser` section below? We should probably change the text there to be explicit about it being `static`, yeah?

Reply:
right now we have the "static" in there:
![image](https://user-images.githubusercontent.com/67650599/123127521-f8e09d80-d40f-11eb-95c2-daa3f181a395.png)
Do you mean the title as in something like "Static HEAL Jupyter Notebooks"?

Reply:
Exactly. I think the point of the changes is that users might think, "These are notebooks I can use", when really they're just static demos/representations/examples. I think updating the title to something like `Static HEAL Jupyter Notebooks` might help make that clear and consistent with the nav text